chart/templates/cleanup/cleanup-serviceaccount.yaml

```diff
@@ -18,11 +18,11 @@
 ################################
 ## Airflow Cleanup ServiceAccount
 #################################
-{{- if and .Values.rbacEnabled .Values.cleanup.enabled }}
+{{- if and .Values.cleanup.serviceAccount.create .Values.cleanup.enabled }}
 kind: ServiceAccount
 apiVersion: v1
 metadata:
-  name: {{ .Release.Name }}-cleanup
+  name: {{ include "cleanup.serviceAccountName" . }}
   labels:
```

Comment:
Is there a reason we don't have the option to add annotations in this service account?

Reply:
We should support annotation for the service account. It is needed for Workload Identity https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity